docs/en/welcome.md

```diff
@@ -0,0 +1,15 @@
+The AI ​​Video Super Resolution solution helps customers reconstruct high-resolution videos from low-resolution videos through self-developed super-resolution algorithms based on deep learning. For example, a 480p video can be converted to a 1080p video. The solution supports private deployment, and allows customers to convert video data in their own accounts.
```

Comment:
we don't provide such information here 'The solution supports private deployment'